CONTRIBUTING.md

```diff
@@ -1,15 +1,99 @@
+  - [Add new comparisons](#add-new-comparisons)
+  - [Consider donation](#consider-donation)
+  - [Talk about Tournesol to your friends](#talk-about-tournesol-to-your-friends)
+- [Got a question or Problem?](#got-a-question-or-problem)
+- [Found a bug?](#found-a-bug)
+- [Missing Feature?](#missing-feature)
+
+**Working on the Source Code**
+
+- [Working on the Source Code](#working-on-the-source-code)
+  - [The development process / Asking for review](#the-development-process--asking-for-review)
+  - [Development guidelines](#development-guidelines)
+    - [Back end](#back-end)
+    - [Front end](#front-end)
+  - [Translation](#translation)
+
+---
+
+## Code of Conduct
+
+> coming soon
+
+## Efficient Ways to Contribute
+
+### Add new comparisons
+
+Everyone can greatly help the project by
+[submitting new comparisons][ts-compare] in the application 🌻
+
+Comparisons are the cornerstones of Tournesol. They are used to provide
+better recommendations and to create an open database to help the research
+on ethics of algorithms and large scale recommender systems.
+
+### Consider making a donation
+
+The Tournesol team is small, mainly composed of volunteers. Every
+donation counts for us.
+
+With more support we can improve the project, by hiring new developers to
+speed up the development of the application, creating partnership to make
+Tournesol more visible on the internet, etc.
+
+If you would like to support us, please refer to [the donation page][ts-donate].
+
+### Talk about Tournesol to your friends
+
+With a bigger community Tournesol can improve its recommendations and the
+quality of its open database.
+
+Tournesol behaves as the algorithmic representative of its community. The more
+the community is big and diverse, the more the recommendation and the database
+will be representative of a larger scale of the population opinion.
+
+Talking about Tournesol to your friends can help the project to transform from
+a small initiative to a broadly known step in the ethics of algorithms and
+large scale recommender systems research.
+
+## Got a Question or Problem?
+
+Please do not open issues for general support questions as we want to keep
+GitHub issues for bug reports and feature requests.
+
+We collect and answer questions on [Discord][ts-discord-join], in the
+dedicated channels. This way all the community can benefit from your question
+and also provides answers.
+
+## Found a Bug?
+
+If you find a bug in the application or in the source code, you can help us by
+submitting an issue to our [GitHub Repository][ts-github-repo]. Even better,
+if you are a programmer with spare time you can submit a Pull Request with a
+fix.
+
+## Missing Feature?
+
+You can request a new feature by submitting an issue to our GitHub Repository.
+Depending on the subject or the complexity, and before planning anything, the
+development team will first discuss the feature to define a clear
+implementation.
```

Comment:
I find the sentence you propose a bit convoluted and missing the mention of "priorities"

```suggestion
Depending on the subject or the complexity the development team will first
evaluate the value of the feature for the whole community, then plan how it
can be implemented and where it fits in the project's priorities.
```